### PR TITLE
Topic: Web Browser

### PR DESCRIPTION
**Pull request by daminimehra**
<hr>

## Description 📜

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. 
Fixes #1523 

<hr>

## Type of change 📝

- [ ] Audio (Should be in mp3 format Includes speech clarity, Concise ,Low distortion)
- [ ] Video (Animations, screen-recordings, presentations and regular explanatory films are all possibilties etc)
- [x] Documentation (Content Creation in the form of codes or tutorials)
- [ ] Other (If you choose other, Please mention changes below) 

<hr>

## Domain of Contribution 📊

- [ ] Android Dev (Flutter, Java, Kotlin)
- [ ] Backend Dev (Java, .NET, Python)
- [ ] C/CPP
- [ ] Competitive Programming
- [ ] Cyber Security
- [ ] DSA
- [ ] Database
- [ ] Datascience with Python
- [ ] Datascience with R
- [ ] Frontend Dev HTML/CSS/JS
- [ ] Frontend Dev React/Angular/Vue
- [ ] Golang
- [ ] Interview Prep
- [ ] Java_Domain
- [x] JavaScript
- [ ] MERN
- [ ] Machine Learning
- [ ] Open Source
- [ ] Python_Domain
- [ ] Rust
- [ ] Statistics
- [ ] UI/UX

<hr>

## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

## Screenshots / Gif (Optional) 📸

<hr>

### CODE REVIEW

**Comment by Creatoon** [comment]
@daminimehra see browser is a runtime and a runtime consits many things, like js engine, call stack and heap, event loop, callback queue and some browser api's like fetch, console(you covered), so also add these things also add the image of browser runtime.

Talk about different browsers and their engines like for chrome - V8 engine and firefox - spidermonkey, just list them for knowledge

And define whole chrome browser runtime keeping all the stuff in mind in a correct order, leave those which you covered

File: JavaScript/Topics/3.Web_Browser/README.md [hunk 1]
```diff
@@ -1 +1,71 @@
...
+    </script>
+</body>
+</html>
+
+```
+
+The result of the above code will look as follows on the HTML Page-
+
+<img title="a title" alt="Alt text" src="img1.png">
+
```
Comment (dakshsethi) [inline JavaScript/Topics/3.Web_Browser/README.md:50]: Crop the image as the rest of the part is not required
Just show the tabs and the thing it is printing
Too much unnecessary white blank space of webpage is not needed

File: JavaScript/Topics/3.Web_Browser/README.md [hunk 2]
```diff
@@ -1 +1,71 @@
...
+However, our main focus in this tutorial is to understand JavaScript. Therefore, we will not do any styling or add CSS here.
+
+We have two options to place the JavaScript in the code. One is in the <head> section and the other is after the <body> tag. But if we view other professional websites, they place their JavaScript at the end of the <body> tag so that the DOM does not get affected. The basic code of JavaScript is as follows-
+
```
Comment (dakshsethi) [inline JavaScript/Topics/3.Web_Browser/README.md:54]: Explain why DOM does not get affected with a paragraph.
Explain why writing the `<script>` tag at the end of `<body>` is a good practice.

File: JavaScript/Topics/3.Web_Browser/README.md [hunk 3]
```diff
@@ -1 +1,71 @@
...
+```js
+//Write your js code here
+console.log("Hello world");
+```
+
+If we open the console section of the browser, we can see the output of the above code as shown below-
+
+<img title="a title" alt="Alt text" src="img2.png">
+
```
Comment (dakshsethi) [inline JavaScript/Topics/3.Web_Browser/README.md:63]: Same here, crop this image too.

**Review by dakshsethi — CHANGES_REQUESTED** [review]

**Comment by daminimehra** [comment]
Sir Pls close this PR. Actually, I have deleted that fork repo and now I am not able to find the location to my branch in file finder and not able to make changes.
I will make new as soon as possible with all the suggested changes by mentors.
@paramsiddharth @dakshsethi

**Comment by paramsiddharth** [comment]
@daminimehra If you find any problems, just let us know. Meanwhile I'll check out your response in Discord.

**Pull request closed without merging**